tests/manage/test_reclaim_policy.py

```diff
@@ -0,0 +1,94 @@
+import logging
+import pytest
+from ocs_ci.framework.testlib import tier1, ManageTest
+from ocs_ci.ocs.resources import pod
+from tests import helpers
+from ocs_ci.ocs import ocp
+
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    This is a test fixture
+    """
+    self = request.node.cls
+
+    def finalizer():
+        teardown(self)
+
+    request.addfinalizer(finalizer)
+    setup(self)
+
+
+def setup(self):
+    """
+    Setting up a secret and storage class
+    """
+    self.secret_obj = helpers.create_secret('CephBlockPool')
+    self.sc_obj_retain = helpers.create_storage_class(
+        interface_type='CephBlockPool', interface_name='rbd',
+        secret_name=self.secret_obj.name, reclaim_policy='Retain'
+    )
+    self.sc_obj_delete = helpers.create_storage_class(
+        interface_type='CephBlockPool', interface_name='rbd',
+        secret_name=self.secret_obj.name, reclaim_policy='Delete'
+    )
+
+
+def teardown(self):
+    """
+    Deleting secret
+    """
+    assert self.sc_obj_retain.delete()
+    assert self.sc_obj_delete.delete()
+    assert self.secret_obj.delete()
+
+
+def list_ceph_images(pool_name='rbd'):
+    """
+    Args:
+        pool_name: (str) Name of the pool to get the ceph images
```

Comment:
change to:
pool_name (str): Name of the pool to get the ceph images

Reply:
Done.